src/backend/app.js

```diff
@@ -21,6 +24,177 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+router.get('/', async (req, res) => {
```

Comment:
app.js shouldn't have this implementation of apis/routes but instead they should be  moved to meals.js and reservation.js